src/web/components/elements/DisclaimerBlockComponent.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { body } from '@guardian/src-foundations/typography';
+import { pillarPalette } from '@root/src/lib/pillars';
+import { css } from 'emotion';
+
+const style = (pillar: Pillar) => css`
+    ${body.small()};
+
+    a {
+        color: ${pillarPalette[pillar].dark};
+    }
+
+    sup {
+        font-size: 12.75px;
```

Comment:
Frontend is `font-size: 75%` which might get you the same but a little less magic.

Reply:
Have resolved.